src/polycubed/src/cube_xdp.cpp

```diff
@@ -44,9 +44,13 @@ CubeXDP::CubeXDP(const std::string &name,
       break;
   }
 
-  // it has to be done here becuase it needs the load, compile methods
+  // it has to be done here because it needs the load, compile methods
   // to be ready
   Cube::init(ingress_code, egress_code);
+  auto devmap_ = master_program_->get_devmap_table("xdp_devmap_");
+  xdp_devmap_ = std::unique_ptr<ebpf::BPFDevmapTable>(new ebpf::BPFDevmapTable(devmap_));
+
+  logger->debug("Created devmap with fd {0}", xdp_devmap_->get_fd());
```

Comment:
Are this kind of debug messages really worthy?

Reply:
Agree, we can definitely remove this debug message.